keel-sql/src/main/resources/db/changelog/20201221-shorten-environment-last-verified-pk.yml

```diff
@@ -0,0 +1,30 @@
+databaseChangeLog:
+- changeSet:
+    id: shorten-environment-last-verified-pk
+    author: fletch
+    changes:
+    - dropAllForeignKeyConstraints:
+        baseTableName: environment_last_verified
+    - delete:
+        tableName: environment_last_verified
+    - dropPrimaryKey:
+        tableName: environment_last_verified
```

Comment:
Need to delete the content of the table as it may violate the shortened primary key. Also, I got an error dropping the PK if I didn't drop the FKs first.